ui/src/app/job-list/table/bulk-edit/bulk-edit.component.html

```diff
@@ -0,0 +1,43 @@
+<h2 mat-dialog-title>Set Metadata</h2>
+<mat-dialog-content>
+  <ng-container *ngFor="let dfItem of data.bulkLabelFields">
+    <div [ngSwitch]="getFieldType(dfItem.displayField)">
+      <div *ngSwitchCase="'text'">
+        <label for="{{ dfItem.displayField.field }}">{{dfItem.displayField.display}}</label>
+        <mat-form-field>
+          <input matInput
+                 type="text"
+                 [value]="(dfItem.default !== false) ? dfItem.default : '(multiple)'"
+                 [id]="dfItem.displayField.field"
+                 #bulkLabelField
+                 maxlength="255"
```

Comment:
How about replacing this number with a variable defined in the component since it's used in a couple of places in this template?